content/.eslintrc

```diff
@@ -5,6 +5,7 @@
     },
     "globals": {
         "Coral": false,
-        "Granite": false
+        "Granite": false,
+        "CMP": false
```

Comment:
We already use the `window.cq.CoreComponents` namespace in other JS files.

Reply:
I think we should use a different NS for libraries which are used for clients on publish so we don't expose the used platform and also the `window.cq` NS is used for `WCM` related stuff.